permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsControllerImpl.kt

```diff
@@ -281,26 +281,19 @@ class PermissionsControllerImpl(
      * @see https://developer.android.com/guide/topics/connectivity/bluetooth/permissions
      */
 
-    private fun allBluetoothPermissions() =
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
-            listOf(
-                Manifest.permission.BLUETOOTH_CONNECT,
-                Manifest.permission.BLUETOOTH_SCAN,
-                Manifest.permission.BLUETOOTH_ADVERTISE,
-                Manifest.permission.ACCESS_COARSE_LOCATION
-            )
-        } else {
-            listOf(
-                Manifest.permission.BLUETOOTH,
-                Manifest.permission.ACCESS_COARSE_LOCATION
-            )
-        }
+    private fun allBluetoothPermissions() = buildSet {
+        addAll(bluetoothConnectCompat())
+        addAll(bluetoothScanCompat())
+        addAll(bluetoothAdvertiseCompat())
+    }.toList()
```

Comment:
This seemed more maintainable (don't need to keep "all" aligned with the other specific permission types, at the expense of allocating a `Set`), let me know if you'd prefer me to use the static lists (as it was before).